NOTICE

```diff
@@ -1,9 +1,2 @@
 OpenSearch
 Copyright 2021 OpenSearch Contributors
```

Comment:
Please refer to [this](https://github.com/opensearch-project/opensearch-plugins/blob/main/NOTICE) for NOTICE file.